src/systems/audio-system.js

```diff
@@ -92,6 +94,16 @@ async function enableChromeAEC(gainNode) {
     await inboundPeerConnection.setRemoteDescription(offer);
 
     const answer = await inboundPeerConnection.createAnswer();
+
+    // Rewrite SDP to be stereo and (variable) max bitrate
+    const parsedSdp = sdpTransform.parse(answer.sdp);
+    for (let i = 0; i < parsedSdp.media.length; i++) {
+      for (let j = 0; j < parsedSdp.media[i].fmtp.length; j++) {
+        parsedSdp.media[i].fmtp[j].config += `;stereo=1;cbr=0;maxaveragebitrate=510000;`;
```

Comment:
Given that we are overriding the decoder stereo preference, maybe we should also override the sender `sprop-stereo` preference? And this is totally a question more than a change request.

Reply:
I already forget how the webrtc negotiation process works since it has fallen out of mental cache, but if I remember correctly its the answer that is going to determine what both the "sender" and "receiver" will do. In any case I confirmed in the webrtc inspector that the bitrate is indeed increased and the output is definitely audibly stereo.